crates/aptos-rest-client/src/lib.rs

```diff
@@ -317,7 +341,7 @@ impl Client {
         let response = self
             .inner
             .post(url)
-            .header(CONTENT_TYPE, BCS_CONTENT_TYPE)
+            .header(CONTENT_TYPE, BCS_SIGNED_TRANSACTION)
```

Comment:
I'd rather we go the other direction and just use BCS_CONTENT_TYPE everywhere. See discussion in comment above.